src/components/ClusterTemplateInstancesTable/ClusterTemplateInstancesTable.tsx

```diff
@@ -55,6 +78,19 @@ const InstanceRow: React.FC<{
       <Td dataLabel={columns[2].title} data-testid="status">
         <ClusterTemplateInstanceStatus instance={instance} />
       </Td>
+      <Td isActionCell>
+        <ActionsColumn
+          items={getRowActions()}
+          actionsToggle={(props: CustomActionsToggleProps) => <KebabToggle {...props} />}
```

Comment:
This can be
```suggestion
          actionsToggle={KebabToggle}
```
right ?